util/cli/config_test.go

```diff
@@ -0,0 +1,80 @@
+package cli
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"testing"
+)
+
+// func TestReadLocalFile(t *testing.T) {
```

Comment:
Please remove commented test

Reply:
Realized that the test was still relevant, so updated it instead.  Thanks for pointing that out!!!